Lib/test/test_dict.py

```diff
@@ -1312,6 +1312,12 @@ def test_reversed(self):
         self.assertEqual(list(r), list('dcba'))
         self.assertRaises(StopIteration, next, r)
 
+    def test_empty_reversed(self):
```

Comment:
Can you add the same test for the non-literal version and for the rest of the iterators:

```
        the_dict = {}
        self.assertEqual(list(reversed(the_dict)), list())
        self.assertEqual(list(reversed(the_dict.values())), list())
        self.assertEqual(list(reversed(the_dict.keys())), list())

        the_dict = dict()
        self.assertEqual(list(reversed(the_dict)), list())
        self.assertEqual(list(reversed(the_dict.values())), list())
        self.assertEqual(list(reversed(the_dict.keys())), list())
```